src/sqlfluff/dialects/dialect_ansi.py

```diff
@@ -2572,10 +2572,13 @@ class FunctionParameterListGrammar(BaseSegment):
 
     # Function parameter list
     match_grammar = Bracketed(
-        Delimited(
-            Ref("FunctionParameterGrammar"),
-            delimiter=Ref("CommaSegment"),
-        )
+        Sequence(
+            Delimited(
+                Ref("FunctionParameterGrammar"),
+                delimiter=Ref("CommaSegment"),
+            ),
+            optional=True,
+        ),
```

Comment:
```suggestion
        Delimited(
            Ref("FunctionParameterGrammar"),
            delimiter=Ref("CommaSegment"),
            optional=True,
        ),
```

Reply:
Now, I feel like I need to study the code base more, didn't know `delimited` support `optional`.